tests/test_vec_envs.py

```diff
@@ -31,14 +31,14 @@ def reset(self, seed: Optional[int] = None):
             self.seed(seed)
         self.current_step = 0
         self._choose_next_state()
-        return self.state
+        return self.state, {}
 
     def step(self, action):
         reward = float(np.random.rand())
         self._choose_next_state()
         self.current_step += 1
-        done = self.current_step >= self.ep_length
-        return self.state, reward, done, {}
+        done = truncated = self.current_step >= self.ep_length
```

Comment:
If i'm not mistaken, in the gymv0.26 API done would be false here as this is not a terminal state, and truncation would be true. (setting only truncation to true would also allow for testing of correct handling of all cases of done and truncation)

Reply:
See my comments in https://github.com/openai/gym/issues/3102#issuecomment-1264648469

My current interpretation of it is made to match gym 0.21 semantic: 
- terminated=True -> reset
- truncated=True -> reset (included in terminated) but bootstrap is valid

> also allow for testing of correct handling of all cases of done and truncation)

we do that in test_gae